stl/inc/iterator

```diff
@@ -1016,15 +1016,20 @@ struct _Common_iterator_pointer_type<_Iter> {
     using pointer = decltype(_STD declval<_Iter&>().operator->());
 };
 

```

Comment:
[P2259R1](http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2021/p2259r1.html) "4. Edit 23.5.4.5 [common.iter.nav] p5 as indicated:" describes changes to `common_iterator`'s `operator++(int)` that don't appear to be implemented above. Compare https://eel.is/c++draft/common.iter.nav#5 to:
https://github.com/microsoft/STL/blob/a9321cfe53ea31a7e197d5d8336167d6ca3de8b6/stl/inc/iterator#L912-L914

Reply:
The changes to `operator++(int)` are implemented in #1991

Reply:
Ah, wonderful! Thanks, I had totally forgotten about that.